lib/alephant/lookup/lookup_query.rb

```diff
@@ -9,7 +9,9 @@ class LookupQuery
       attr_reader :table_name, :lookup_location
 
       def initialize(table_name, component_id, opts, batch_version)
-        @client          = AWS::DynamoDB::Client::V20120810.new
+        options = {}
```

Comment:
See the comments from your other PRs

copy paste follows:
Is there a specific reason for the `merge`?
`options = ENV['AWS_S3_ENDPOINT'] ? {endpoint: ENV['AWS_S3_ENDPOINT']} : {}` perhaps?
or if you still want it separated out
```
options = {}
options[:endpoint] = ENV['AWS_S3_ENDPOINT'] if ENV['AWS_S3_ENDPOINT']
```